go.mod

```diff
@@ -1,36 +1,33 @@
 module github.com/gobuffalo/pop/v5
 
-go 1.15
+go 1.16
 
-replace github.com/mattn/go-sqlite3 => github.com/mattn/go-sqlite3 v1.14.7
+replace github.com/mattn/go-sqlite3 => github.com/mattn/go-sqlite3 v1.14.9
```

Comment:
@paganotoni  This replace statement causes quite a bit of trouble and should be removed. This requires a temporary downgrade to `fizz` `v1.10.0`.

`fizz` can then be updated to use the newer version of pop without the `replace` statement, fixing the dependency graph of `fizz`.

Lastly this the new `fizz` version can be used here without the `replace` statement

I mention this issue in my PR: https://github.com/gobuffalo/pop/pull/667

Reply:
Hi @fasmat,

I didn't know you are preparing PR for pop when I requested this PR. Please ignore this PR and please go ahead with your #667! I will close this PR for now!